{
  "id": "CVE-2018-6552",
  "assigner": "security@ubuntu.com",
  "Published": "2018-05-31T20:29:00.207Z",
  "Modified": "2019-10-02T22:03:26.223Z",
  "last-modified": "2019-10-02T22:03:26.223Z",
  "summary": "Apport does not properly handle crashes originating from a PID namespace allowing local users to create certain files as root which an attacker could leverage to perform a denial of service via resource exhaustion, possibly gain root privileges, or escape from containers. The is_same_ns() function returns True when /proc/<global pid>/ does not exist in order to indicate that the crash should be handled in the global namespace rather than inside of a container. However, the portion of the data/apport code that decides whether or not to forward a crash to a container does not always replace sys.argv[1] with the value stored in the host_pid variable when /proc/<global pid>/ does not exist which results in the container pid being used in the global namespace. This flaw affects versions 2.20.8-0ubuntu4 through 2.20.9-0ubuntu7, 2.20.7-0ubuntu3.7, 2.20.7-0ubuntu3.8, 2.20.1-0ubuntu2.15 through 2.20.1-0ubuntu2.17, and 2.14.1-0ubuntu3.28.",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "LOCAL"
  },
  "impact": {
    "availability": "COMPLETE",
    "confidentiality": "COMPLETE",
    "integrity": "COMPLETE"
  },
  "cvss": 7.2,
  "exploitabilityScore": 3.9,
  "impactScore": 10.0,
  "cvss-vector": "AV:L/AC:L/Au:N/C:C/I:C/A:C",
  "references": [
    "https://usn.ubuntu.com/3664-2/",
    "https://usn.ubuntu.com/usn/usn-3664-1"
  ],
  "vulnerable_configuration": [],
  "vulnerable_product": [],
  "vendors": [],
  "products": [],
  "vulnerable_product_stems": [],
  "vulnerable_configuration_stems": [],
  "cwe": "NVD-CWE-noinfo",
  "vulnerable_configuration_cpe_2_2": []
}